src/Features/Core/Portable/ExternalAccess/UnitTesting/API/IUnitTestingStackTraceServiceAccessor.cs

```diff
@@ -13,6 +13,6 @@ internal interface IUnitTestingStackTraceServiceAccessor : IWorkspaceService
 {
     Task<ImmutableArray<UnitTestingParsedFrameWrapper>> TryParseAsync(string input, Workspace workspace, CancellationToken cancellationToken);
     Task<UnitTestingDefinitionItemWrapper?> TryFindMethodDefinitionAsync(Workspace workspace, UnitTestingParsedFrameWrapper parsedFrame, CancellationToken cancellationToken);
-    (Document? document, int lineNumber) GetDocumentAndLine(Workspace workspace, UnitTestingParsedFrameWrapper parsedFrame);
+    (TextDocument? document, int lineNumber) GetDocumentAndLine(Workspace workspace, UnitTestingParsedFrameWrapper parsedFrame);
```

Comment:
Is this going to be a binary breaking change for the external access layer? I'd assume changing the return type like this might cause missing method exceptions.

Reply:
oh it is. Just quickly going through and missed that this was EA